crates/bins/wafl/src/commands/rpc.rs

```diff
@@ -23,13 +24,9 @@ pub(crate) enum SubCommands {
 
 #[derive(Debug, Clone, Args)]
 pub(crate) struct ConnectOptions {
-  /// RPC port.
-  #[clap(short, long, env = wasmflow_collection_cli::options::env::WAFL_RPC_PORT,action)]
-  pub(crate) port: u16,
-
-  /// RPC address.
-  #[clap(short, long, default_value = "127.0.0.1", env = wasmflow_collection_cli::options::env::WAFL_RPC_ADDRESS,action)]
-  pub(crate) address: String,
+  /// RPC Url
+  #[clap(short, long)]
+  pub(crate) url: Url,
```

Comment:
Here I'm missing some validation so Im able to `unwrap` with confidence on L50 and L54.